src/components/ABDM/ABDMFacilityRecords.tsx

```diff
@@ -184,6 +184,13 @@ export default function ABDMFacilityRecords({ facilityId }: IProps) {
                 </table>
               </div>
             </div>
+            {consentsResult?.results.length === 0 && (
+              <div className="flex min-h-[200px] items-center justify-center p-4">
```

Comment:
avoiding hardcoding px values. use rem values. behaves differently for different zoom sizes.

Reply:
Added the changes